client/src/store/modules/eventType/actions.js

```diff
@@ -40,8 +40,6 @@ export default {
             const eventType = await eventTypeService.addEventType(
                 eventTypeFormMapper(data)
             );
-            context.commit(mutations.SET_EVENT_TYPES, eventType);
```

Comment:
I am not sure if this should be removed. Was it accidentally removed?

Reply:
that mutation is not needed here, 
she is not working as we need, and when we redirected to 'EventTypes' page, all eventTypes is loaded from backend, there is no need for it.